geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -127,6 +127,11 @@ class AccumuloDataStore(val connector: Connector,
           // do this before anything else so that any modifications will be in place
           GeoMesaSchemaValidator.validate(sft)
 
+          // TODO GEOMESA-1322 support tilde in feature name
```

Comment:
I have a slight preference for a ticket for the exception and one for the fix...